examples/1-simple-pipeline.yaml

```diff
@@ -3,6 +3,8 @@ kind: Pipeline
 metadata:
   name: simple-pipeline
 spec:
+  watermark:
```

Comment:
I would be better to create a new file particularly with watermark turned on, and leave this one for simple use case demonstration.

Reply:
Sorry, I had merged this by mistake. Will remove it :) and add a separate one for watermark

Reply:
Added an example for watermark

Reply:
let's not add watermark example yet. we need to iron few more details

Reply:
ok, removed it